engines/router/missionctl/testdata/upi_small_payload.json

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
nit: Rather than hardcoding the payload shall we make a utility function to programmatically create one?